core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -66,6 +65,9 @@ private[spark] class CoarseMesosSchedulerBackend(
 
   val extraCoresPerSlave = conf.getInt("spark.mesos.extra.cores", 0)
 
+  // Offer constraints
+  val slaveOfferConstraints = parseConstraintString(sc.conf.get("spark.mesos.constraints", ""))
```

Comment:
@ankurcha I think you missed this comment in the update.  (unless there is some reason it can't be `private` that I'm missing?)